pkg/plugin/v3/init.go

```diff
@@ -40,8 +40,9 @@ type initPlugin struct {
 	commandName string
 
 	// boilerplate options
-	license string
-	owner   string
+	license         string
+	owner           string
+	componentConfig bool
 
 	// flags
 	fetchDeps          bool
```

Comment:
Place it with the flags instead of with the boilerplate options
```suggestion
	license string
	owner   string

	// flags
	componentConfig    bool
	fetchDeps          bool
```